### PR TITLE
Create evaluation readme

### PR DESCRIPTION
**Pull request by kathsherratt**
This PR visualises ensemble performance in:
- a README, modified from ensemble evaluation presentation (code by @sbfnk plus a couple of figure captions)
- run via github action workflow, each Tuesday and Wednesday (similar to the timing of the website model reports)

### CODE REVIEW

File: code/evaluation/README.Rmd [hunk 1]
```diff
@@ -92,7 +92,33 @@ criteria <- bind_rows(criteria, .id = "ensemble_date") |>
   mutate(ensemble_date = as.Date(ensemble_date))
 ```
 
-## 50% Coverage
+## Number of models in ensemble
```
Comment (sbfnk) [inline code/evaluation/README.Rmd:95]:
```suggestion
# Data files of forecast scores

The files `scores.csv` contains "raw" scores for each data/forecast point based on the predictive quantiles and where the data ended up relative to them. For each combination of date, target variable and location we report the [Weighted Interval Score](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008618) (`wis`) as well as its components of `sharpness`, `underprediction` and `overprediction`, the absolute error of the median (`aem`), [Bias](https://doi.org/10.1371/journal.pcbi.1006785) (`bias`), coverage at the 50 and 95% levels (`cov_50`/`cov_95`), and the number of quantiles supplied (`n_quantiles`).

The `weekly-summary` folder contains weekly evaluations that enter the [weekly performance reports](https://covid19forecasthub.eu/reports.html).

# Ensemble performance
## Number of models in ensemble
```

File: .github/workflows/render-readme-evaluation.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+      - name: Install system dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install libudunits2-dev libcurl4-openssl-dev libgdal-dev
+
+      - uses: r-lib/actions/setup-renv@v2
+
+      - name: Render README
+        run: Rscript -e 'rmarkdown::render("code/evaluation/README.Rmd")'
```
Comment (sbfnk) [inline .github/workflows/render-readme-evaluation.yml:35]:
```suggestion
        run: Rscript -e 'rmarkdown::render("evaluation/README.Rmd")'
```

File: .github/workflows/render-readme-evaluation.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+      - name: Commit results
+        run: |
+          git config --local user.email "actions@github.com"
+          git config --local user.name "GitHub Actions"
+          git commit README.md -m 'Re-build evaluation README.Rmd' || echo "No changes to commit"
```
Comment (sbfnk) [inline .github/workflows/render-readme-evaluation.yml:41]: Does this have to be:

```suggestion
          git commit evaluation/README.md -m 'Re-build evaluation README.Rmd' || echo "No changes to commit"
```

File: .github/workflows/render-readme-evaluation.yml [hunk 4]
```diff
@@ -0,0 +1,42 @@
+on:
+  schedule:
+    - cron: "0 13 * * 2,3"
+  push:
+    paths:
+      - code/evaluation/README.Rmd
```
Comment (sbfnk) [inline .github/workflows/render-readme-evaluation.yml:6]:
```suggestion
      - evaluation/README.Rmd
```

**Review by sbfnk — CHANGES_REQUESTED** [review]
Thanks, looks great! A few quick suggestions:
- put it in `evaluation/` rather than `code/evaluation` - my bad for pointing to the wrong place, sorry! I've added the text from the existing README there
- maybe edit action here instead of adding a new one? Makes no difference in practice but might be preferable to have less code https://github.com/covid19-forecast-hub-europe/covid19-forecast-hub-europe/blob/0d3e120f4af6a01a7b221a6746ac6b2d75f65380/.github/workflows/scoring.yml#L33
- can you also render once and commit just so we can check what the plots look ilke?

**Comment by kathsherratt** [comment]
Thanks! Implemented those suggestions. Please merge if you think it's ready (of course no problem to edit).

**Review by sbfnk — APPROVED** [review]

**Pull request merged**